src/errors/request.ts

```diff
@@ -28,10 +28,10 @@ export class HttpRequestError extends BaseError {
       metaMessages: [
         status && `Status: ${status}`,
         `URL: ${getUrl(url)}`,
-        `Request body: ${stringify(body)}`,
+        body && `Request body: ${stringify(body)}`,
       ].filter(Boolean) as string[],
     })
-    this.body = body
+    if (body) this.body = body
```

Comment:
`body` `undefined` if not provided so seems correct the way it was before.

```suggestion
    this.body = body
```